### PR TITLE
Personalized Tokens

### PR DESCRIPTION
**Pull request by Magnus-Kuhn**
# Readiness checklist

-   [X] I added/updated tests.
-   [X] I ensured that the PR title is good enough for the changelog.
-   [X] I labeled the PR.

### CODE REVIEW

**Comment by jkoenig134** [comment]
I don't see tests for tokens for relationship templates. Was this added in the other PR?

**Comment by Magnus-Kuhn** [comment]
> I don't see tests for tokens for relationship templates. Was this added in the other PR?

The scenario of a personalized token for a non-personalized template was not there - I'm adding it in

**Review by jkoenig134 — APPROVED** [review]
LGTM. @Magnus-Kuhn please wait for feedback by minimum @britsta or @Milena-Czierlinski

**Review by Milena-Czierlinski — DISMISSED** [review]

File: test/tokens.test.ts [hunk 1]
```diff
@@ -12,16 +13,48 @@ let client2: ConnectorClient;
 beforeAll(async () => ([client1, client2] = await launcher.launch(2)), getTimeout(30000));
 afterAll(() => launcher.stop());
 
+test("send a token", async () => {
+    const response = await client1.tokens.createOwnToken({
+        content: {
+            content: "Hello"
```
[the remark this answers is not in the document]
Comment (Magnus-Kuhn) [reply test/tokens.test.ts:19]: I copy-pasted this from the test-utils 😆

**Review by jkoenig134 — APPROVED** [review]

File: test/tokens.test.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (jkoenig134) [reply test/tokens.test.ts]: Yeah, solid argument.

**Pull request merged**